core/common/src/main/java/alluxio/worker/block/BlockWorker.java

```diff
@@ -136,13 +136,11 @@ String createBlock(long sessionId, long blockId, int tier,
       throws BlockAlreadyExistsException, WorkerOutOfSpaceException, IOException;
 
   /**
-   * @param sessionId the id of the session to get this file
    * @param blockId the id of the block
-   *
    * @return metadata of the block or null if the temp block does not exist
    */
   @Nullable
-  TempBlockMeta getTempBlockMeta(long sessionId, long blockId);
```

Comment:
I actually not sure we want to change the interface here, we better just update the place we invoke the method?

Reply:
many places, we do not have a reference to the sessionId, that's why I changed it..